packages/react-native/React/Views/UIView+React.m

```diff
@@ -12,6 +12,7 @@
 #import "RCTAssert.h"
 #import "RCTLog.h"
 #import "RCTShadowView.h"
+#import "RCTView.h"
```

Comment:
I think this might cause a circular dependency problem. OK since we already have some and don't support the `use_frameworks` option, but might be something we'll want to undo later.

Also, macOS tag